### ISSUE TITLE
Network stalled for 30 minutes on block 4696

### ISSUE THREAD

**Issue by karek314** [LiskHQ/lisk#152]
Very serious issue happened after launch. Network stucked on block 4696. It happened on my several independent nodes, also some other people confirmed that. Issue must be located in that case. Since most likely it's related to forging 101 delegates and all of theme are currently in management of Lisk Team during launch, please explain if there was ddos or any connectivity issues which may explain source of the problem
<img width="1413" alt="screen shot 2016-05-25 at 14 21 57" src="https://cloud.githubusercontent.com/assets/9548618/15540087/143205ba-2286-11e6-93f3-a0b3522af1f3.png">
Screenshot confirms timestamp difference and queued transactions

**Comment by fix** [comment]
Hi @karek314 
Yes there was an issue on the forging nodes. In this case we have taken a strong security measure in order to prevent forking, that stop forging and wait for the delegate to sync properly. There was a second shortest stop a few minutes later. The transactions were queued so they were released when the delegates retarted forging.

As for reason, it is unclear. Looking at the nodes they were under heavy load. But we have not yet looked into details to state it was a DDoS. This was happening during the major front-end DDoS where literraly terabytes were flooding... On the other ends we are still experiencing issues with the 101 azures VPS, so it looks like that the azure cloud has some issues or does not support well the way we deployed it.

We are working to improve the resynchronisation of all the delegates when many are not forging.
